lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -107,6 +107,16 @@ def store(credit_card, options={})
         end
       end
 
+      def unstore(options={})
+        requires!(options, :shopper_reference)
+        post = init_post(options)
+
+        post[:shopperReference] = options[:shopper_reference]
```

Comment:
Does `shopperReference` allow to disable specific payment methods or would it disable all stored PMs associated with the shopper.

Reply:
It disables all stored PMs associated with the shopper.

Reply:
This is now updated to include the `recurringDetailReference` as it has been determined that it does indeed map, one-to-one, to a payment method.